src/libraries/Common/src/System/IO/ReadOnlyMemoryStream.cs

```diff
@@ -50,10 +50,7 @@ public override long Position
             set
             {
                 EnsureNotClosed();
-                if (value < 0 || value > int.MaxValue)
-                {
-                    throw new ArgumentOutOfRangeException(nameof(value));
-                }
+                ArgumentOutOfRangeException.ThrowIfNotBetween(value, 0, int.MaxValue);
```

Comment:
ThrowIfNotBetween suggests an exclusive range, but the code here treated the ends of the range as valid. Should it be something like ThrowIfOutsideRange?

Reply:
Initially discussed here 
https://github.com/dotnet/runtime/issues/69590#issuecomment-1132958997

But can be easily changed to `ThrowIfOutsideRange`. Added this to `questions` section in issue

Reply:
ThrowIfNotBetween is perhaps a good name, but i think it would not be the method you use here. Anyway, as you say, naming can be discussed at review.